### DISCUSSION TITLE
Can’t  create new project items on iPadOS (safari or chrome browsers)

### DISCUSSION THREAD

**Discussion by PrasadBabarendaGamage** [community/community#65848] [Projects and Issues] [answered]
### Select Topic Area

Bug

### Feature Area

Projects

### Body

Hi, could someone please check to see if they can create new project items (e.g. Kanban board items on a project board) on iPadOS? On a desktop browser, you can add items using the + button at the bottom of the browser screen. However, on Safari and Chrome browsers on iPadOS, the + button does not appear at the bottom of the screen. 

This problem makes it impossible to use GitHub projects on iPadOS even from the browser (the GitHub app does not provide a way to create new items on the project board either).

I have admin rights for the organisation/project board that I am testing with. Changing the zoom level on the browser does not allow the + button to show.

I have the GitHub app installed, and when going to GitHub on a browser, it displays a ribbon at the top of the browser page saying “Open in the GitHub app”. Perhaps its pushing the + button for adding new issues off the page??

Tested using latest version of iPadOS - 16.6.

Thanks

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by DirkLachowski** [comment, marked as answer]
Same here. GitHub app isn’t installed but no “add” button on the board. As a workaround, you can:
- use a list view where adding works on ipadOS
- Move the tab bar of the mobile browser (not the browser content! Just tap and hold on any tab‘s title) slightly up until the tabbar is hidden. That will reveal the add button on mobile safari in card view. Maybe the window height computation is off by the tabbar’s height?

**Reply by FeasterFamine** [reply]
Thanks for the workarounds, this still isn’t working in either ios browser or the app.

**Comment by PrasadBabarendaGamage** [comment]
Thanks @DirkLachowski - your suggested work around seems to work. Pity that Github developers aren’t monitoring these bug reports so that the underlying issue can be fixed.
